addons/custom-block-text/userscript.js

```diff
@@ -0,0 +1,93 @@
+import { updateAllBlocks } from "../custom-block-shape/update-all-blocks.js";
+import { clearTextWidthCache } from "../middle-click-popup/module.js";
+
+export default async function ({ addon, console }) {
+  let currentTextSize = 100;
+
+  const vm = addon.tab.traps.vm;
+  const blocklyInstance = await addon.tab.traps.getBlockly();
+
+  // Handling the CSS from here instead of a userstyle is much more stable, as
+  // there's no code outside of this addon dynamically toggling the styles.
+  // This way, we can clearly control the execution order of style operations.
+  // For example, we always want to call updateAllBlocks() after the styles
+  // were updated according to the user's settings, not before.
+  const fontSizeCss = document.createElement("style");
+  fontSizeCss.textContent = `
+    .blocklyText,
```

Comment:
```suggestion
    .blocklyText:not(.blocklyFlyoutButton > .blocklyText),
```

This change would make the CSS not affect the font size of the "make a variable", "make a block", etc. buttons.

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/682ad543-9c56-4664-879e-af545cb92b76)

Reply:
I think we decided against this.